code/_globalvars/lists/flavor_misc.dm

```diff
@@ -35,6 +35,7 @@ var/global/list/female_clothing_icons = list()
 
 	//radical shit
 var/list/hit_appends = list("-OOF", "-ACK", "-UGH", "-HRNK", "-HURGH", "-GLORF")
+var/list/extraction_appends = list("-AAAAAAAAAAAAAAAAAUGH", "-AAAAAAAAAAAHHHHHHHHHH")

```

Comment:
figured I'd put it with the other appends because it used the same system and shit